redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -312,15 +312,15 @@ internal class LazyListContainerCell(
 
     if (contentView.subviews.isEmpty()) return
 
-    val content = contentView.subviews.first() as UIView ?: return
+    val content = contentView.subviews.firstOrNull() as UIView ?: return
```

Comment:
```suggestion
    val content = contentView.subviews.firstOrNull() as UIView? ?: return
```
So without this the cast will throw on null. We need to cast to a nullable version of the type we expect so that null can pass through.

Reply:
Ooooh . . . I think we can skip the `isEmpty()` check above if we use `firstOrNull()` here.

Reply:
Much cleaner, thanks!